.circleci/config.yml

```diff
@@ -44,7 +46,7 @@ jobs:
           name: Build & push docker image
           # NOTE: the env variables holding docker credentials are stored in the CircleCI dashboard
           command: |
-            git diff --name-only HEAD^ | grep -E '(Dockerfile|trellis)' || exit 0
+            git diff --name-only origin/HEAD | grep -E '(Dockerfile|trellis)' || exit 0
```

Comment:
Diff against `origin`, not `HEAD`'s parent. This will allow our setup to trigger image updates with multi-commit changes to `master`.